packages/vite-plugin-cloudflare/src/miniflare-options.ts

```diff
@@ -159,8 +159,11 @@ export function getMiniflareOptions(
 	return {
 		log: logger,
 		handleRuntimeStdio(stdout, stderr) {
-			stdout.forEach((data) => logger.info(data));
-			stderr.forEach((error) => logger.error(error));
+			const decoder = new TextDecoder();
+			stdout.forEach((data) => logger.info(decoder.decode(data)));
+			stderr.forEach((error) =>
+				logger.logWithLevel(LogLevel.ERROR, decoder.decode(error)),
+			);
```

Comment:
just out curiosity, is the `TextDecoder` necessary? yesterday I tried with `data.toString()` and `error.toString()` and that seemed to do the job 🤔

Reply:
This is just how I have always converted bytes to strings.